wazo_router_confd/schemas/carrier.py

```diff
@@ -9,20 +9,20 @@
 class Carrier(BaseModel):
     id: int
     name: constr(max_length=256)  # type: ignore
-    tenant_uuid: UUID4
+    tenant_uuid: Optional[UUID4]
```

Comment:
Why is the `tenant_uuid` optional?

Reply:
Because it is not mandatory to provide tenant_uuid when using the API, as it will be obtained from the JWT token. If I don't mark it as optional, the user of the API has to always provide a tenant_uuid when consuming the end-points. In any case, each method is trying to obtain the tenant_uuid from the JWT token, if available, falling back on the value provided in the request body. If they don't match, or no value is provided, the API returns error.